Form-Controls/styles.css

```diff
@@ -0,0 +1,198 @@
+
+/*// <uniquifier>: Use a unique and descriptive class name
+// <weight>: Use a value from 100 to 900
+
+.azeret-mono-<uniquifier> {
+  font-family: "Azeret Mono", monospace;
+  font-optical-sizing: auto;
+  font-weight: <weight>;
+  font-style: normal;
+}
+// <uniquifier>: Use a unique and descriptive class name
+// <weight>: Use a value from 100 to 900
+
+.inter-<uniquifier> {
+  font-family: "Inter", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: <weight>;
+  font-style: normal;
+} */
+
+*{
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    background-color: #edeae8;
+}
+
+.logo{
+    height: 100px;
+    width: auto;
+}
+
+.menu {
+ display: flex;
+    justify-content: center;
+ align-items: center;
+ background-color: #edeae8;
+ padding: 30px 50px;
+ margin-bottom: 30px;
+ margin-top: 10px;
+}
+
+header{
+    justify-content: center;
+    text-align:  center;
+    margin-bottom: 10px;
+    background-color: rgb(248, 245, 245);  
+}
+
+h1{
+    padding-top: 40px;
+    margin-top: 40px;
```

Comment:
We typically avoid top margins in layout. This is because as pages get more complicated, you will have many components interacting. In CSS margins can collapse and you can lose control of your spacing

https://www.joshwcomeau.com/css/rules-of-margin-collapse/

Reply:
This is a very good explanation. I saved it on my favourites to future check. 
Solved; took off the paddings and added height to control the elements inside the section